specs-go/config.go

```diff
@@ -353,11 +353,11 @@ type DeviceCgroup struct {
 	Access *string `json:"access,omitempty"`
 }
 
-// Seccomp represents syscall restrictions
-type Seccomp struct {
-	DefaultAction Action    `json:"defaultAction"`
-	Architectures []Arch    `json:"architectures"`
-	Syscalls      []Syscall `json:"syscalls,omitempty"`
+// LinuxSeccomp represents syscall restrictions
+type LinuxSeccomp struct {
+	DefaultAction Action         `json:"defaultAction"`
```

Comment:
You probably want to namespace `Action` → `LinuxSeccompAction` (or maybe `SeccompAction` if we don't namespace `Seccomp`).  `Operator` and `Arg` are in the same boat.